### PR TITLE
Kubernetes: print kernel version during E2E

### PR DESCRIPTION
**Pull request by jackfrancis**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```
NONE
```

### CODE REVIEW

File: test/e2e/kubernetes/kubernetes_test.go [hunk 1]
```diff
@@ -72,6 +72,15 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
...
 				log.Printf("Error while getting Ubuntu image version: %s\n", out)
 			}
+
+			kernelVerCmd := fmt.Sprintf("cat /proc/version")
+			cmd := exec.Command("ssh", "-i", sshKeyPath, "-o", "ConnectTimeout=10", "-o", "StrictHostKeyChecking=no", "-o", "UserKnownHostsFile=/dev/null", master, kernelVerCmd)
+			util.PrintCommand(cmd)
+			out, err := cmd.CombinedOutput()
+			log.Printf("%s\n", out)
+			if err != nil {
+				log.Printf("Error while getting LinuxKernel version: %s\n", out)
```
Comment (mboersma) [inline test/e2e/kubernetes/kubernetes_test.go]: We already logged the `CombinedOutput` when we checked the error value. Maybe this should print the `err` instead?

Comment (jackfrancis) [reply test/e2e/kubernetes/kubernetes_test.go]: Good catch! Corrected, like, in a lot of places!

**Pull request merged**